RespiratoryAssistDevices/R4/resources/Questionnaire-R4-RespiratoryAssistDevices.json

```diff
@@ -270,71 +78,47 @@
           "type": "open-choice",
           "text": "Patient diagnoses for order",
           "required": false,
+          "repeats": true,
           "extension": [
             {
               "url": "http://hl7.org/fhir/StructureDefinition/questionnaire-answerRepeats",
               "valueBoolean": true
-            }
-          ],
-          "answerOption": [
-            {
-              "valueCoding": {
-                "code": "Restrictive thoracic disorder"
-              }
-            },
-            {
-              "valueCoding": {
-                "code": "Severe COPD"
-              }
-            },
-            {
-              "valueCoding": {
-                "code": "Hypoventilation syndrome"
-              }
-            },
-            {
-              "valueCoding": {
-                "code": "Central sleep apnea"
-              }
-            },
-            {
-              "valueCoding": {
-                "code": "Complex sleep apnea"
-              }
-            },
-            {
-              "valueCoding": {
-                "code": "Obstructive sleep apnea (OSA)"
-              }
             },
             {
-              "valueCoding": {
-                "code": "Other"
+              "url": "http://hl7.org/fhir/StructureDefinition/cqf-expression",
+              "valueExpression": {
+                "language": "text/cql",
+                "expression": "\"RespiratoryAssistDevicesPrepopulation\".OsaDiagnoses"
               }
             }
-          ]
+          ],
+          "answerValueSet": "http://cts.nlm.nih.gov/fhir/ValueSet/2.16.840.1.113762.1.4.1219.122"
         },
         {
           "linkId": "4.2",
-          "text": "If Other, describe",
-          "type": "string",
-          "required": true,
-          "enableWhen": [
+          "text": "Other Diagnoses:",
```

Comment:
Will this be changed based on Tuesday's discussion?

Reply:
It should be updated.